src/core/stdlib.js

```diff
@@ -260,6 +260,12 @@ foam.LIB({
       if ( ! a || ! b ) return false;
       if ( a.equals ) return a.equals(b);
       return a == b;
+    },
+    function compare(a, b) {
+      if ( a.compareTo ) return a.compareTo(b);
```

Comment:
I'm not sure. I don't think we did. They're slow, so it's probably better to add `typeof a === "number"` checks to `equals` and `compare` to handle this.

Kevin, thoughts?